support-models/src/main/scala/com/gu/support/catalog/FulfilmentOptions.scala

```diff
@@ -1,13 +1,29 @@
 package com.gu.support.catalog
 
-sealed trait FulfilmentOptions[+T <: Product]
```

Comment:
I've removed the relationship between a FulfilmentOption and a Product because unfortunately it didn't play well with Circe serialisation, although it did allow us to check at compile time that any given fulfilment option was appropriate for a given product